portfolio/src/main/webapp/hiddentalent.html

```diff
@@ -25,22 +25,29 @@ <h1>My Hidden Talent</h1>
         <source src="/audio/Highest in the Woods Instrumental.wav">
         Your browser does not support the audio tag.
       </audio>
-      <form action="/data" method="POST">
-        <p><b>Want to Leave a Comment?</b>
-        <br/>
-        <label for="name">Name:</label>
-        <input type="text" name="name">
-        <br/>
-        <label for="email">Email:</label>
-        <input type="email" name="email">
-        <br/>
-        <label for="comment">Comment:</label>
-        <textarea name="comment"></textarea>
-        <br/><br/>
-        <input type="submit" />
-      </form>
+      <nav id="divider">
+          <li>_
+      </nav>
+      <p><b>Want to Leave a Comment?</b>
+      <br/>
```

Comment:
[Don't close void elements](https://google.github.io/styleguide/htmlcssguide.html#Document_Type). Here, and throughout.